apps/drupal/templates/block/block--system-branding-block.html.twig

```diff
@@ -8,25 +8,17 @@
  * enabled in the block configuration.
  *
  * Available variables:
+ * - url: The url for the front page of the site.
  * - site_logo: Logo for site as defined in Appearance or theme settings.
+ * - logo_inline_svg: The svg logo if it exists.
+ * - logo_img_src: The src logo if it exists.
  * - site_name: Name for site as defined in Site information settings.
  * - site_slogan: Slogan for site as defined in Site information settings.
  */
 #}
 
-
 {% block content %}
 
-  <h1>i r branding nerp</h1>
-
-  {% include '@atoms/button/_button.twig' with {
-    button_size: 'lg',
-    button_text: 'hello',
-  } %}
+  {% include "@molecules/_branding.twig" %}
```

Comment:
No `with{}` here?

Reply:
I can add it for demo purposes, but the logo atom itself is set via the preprocess so it's not really necessary. Plus taking into account the possibility of a duplicate logo I left it out.